src/client/pps/pps.proto

```diff
@@ -112,6 +112,9 @@ message ResourceSpec {
   // The amount of memory, in bytes, each worker needs (in bytes, with allowed
   // SI suffixes (M, K, G, Mi, Ki, Gi, etc).
   string memory = 2;
+
+  // The number of GPUs each worker needs.
+  int64 gpu = 3;
```

Comment:
Drive-by comment: Does k8s support partial GPU sharing? If so, should we make this a float?